### DISCUSSION TITLE
All Features are not showing that available in the demo

### DISCUSSION THREAD

**Discussion by abhinictjsr** [GrapesJS/grapesjs#4121] [Q&A]
All Features are not showing that available in the demo  file that we need 
demo features shows and install features screen shot is given below please help regarding this

Demo Features:-
![image](https://user-images.githubusercontent.com/64882161/152352872-8d4e1ff2-a05f-47fe-9576-cbee0bdc34dd.png)

My Install Features
![image](https://user-images.githubusercontent.com/64882161/152353070-38400b9c-6375-4599-81a5-4ae659792d13.png)

**Comment by artf** [comment]
The demo includes also other plugins and configurations, here you can check the code: https://github.com/artf/grapesjs/blob/gh-pages/demo.html
